torch/testing/_internal/common_methods_invocations.py

```diff
@@ -4792,6 +4792,12 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
            sample_inputs_func=sample_inputs_pow,
            supports_inplace_autograd=False,
            assert_autodiffed=True),
+    OpInfo('float_power',
+           dtypes=all_types_and_complex_and(torch.half, torch.bfloat16, torch.bool),
+           backward_dtypes=all_types_and_complex_and(torch.half, torch.bfloat16, torch.bool),
```

Comment:
Unlike `pow` using `torch.half` shouldn't be a problem since the tensor will be casted to either double/complex double depending on the exponent/base.

Reply:
Since the `backward_dtypes` are same as `dtypes`, you don't need to add `backward_dtypes` field as it defaults to `dtypes` if not passed.

https://github.com/pytorch/pytorch/blob/c129ab780e963adde8bb1f1a8966bd5683c88144/torch/testing/_internal/common_methods_invocations.py#L210

Reply:
Oh, I see. Thanks for pointing this out, @kshitij12345. This is now fixed.